submissions.html

```diff
@@ -0,0 +1,185 @@
+
+Check <strong>line 20</strong> of the template for the line reading <tt>\usepackage[blind]{anonymize}</tt><br>
+
+This is the command that imports the anonymization functionality, and allows the submitter to censor content by wrapping the text around the <tt>\anonymize{ANY CONTINUOUS TEXT HERE}</tt> environment. Notice that in the template, parts of the abstract and the sponsorship footnote were written this way.
+
+<strong> Switch the anonymization on/off:</strong> While writing the paper, you can switch the anonymization off by removing the <tt>[blind]</tt> argument from the command: <tt>\usepackage{anonymize}</tt>.
+
+<br><br> In case there are any questions, please do not hesitate to reach out to the organizers through our <a id="general-enquiry"><u>email</u></a> (quantummusicsubmission AT gmail DOT com)
+
+<p class="font-medium">Music Submission</p>
+<p>The music submission for ISQCMC '25 is currently <u><a id="music-submission"><span style='color: #03CEA4;'>open</span></a></u>. Please refer to the detailed instructions below to ensure a smooth review and avoid unnecessary rejections of your submission.</p>
+
+The deadline for submission is <strong>To Be Announced</strong>. <br>
+<p>Music Submission Instructions</p>
+Please note that Music Submissions are <strong>not anonymized</strong>.<br><br>
+
+All submissions are to be sent to the <a id="music-2submission"><u>quantummusicsubmission AT gmail DOT com</u></a>, and <strong> should contain the following information and files:</strong>
+
+<ul>
+  <li>Full names of all authors, and their respective affiliations (if applicable)</li>
+  <li>Clear indication of the corresponding author(s) and their contact email(s)</li>
+  <li>The manuscript of your work, in PDF format (see manuscript instructions below)</li>
+  <li>Any additional supporting information or appendices</li>
+  <li><strong>For fixed-media compositions -</strong> A stereo audio file (WAV/MP3) or a video file (MP4) with the piece should be submitted along with the manuscript, even if the composition is unfinished at the stage of the first submission (Complete works are preferred). The completed piece will be required for the final submission, in case the work is accepted.</li>
+  <li><strong>For live performances -</strong> Please ensure that a simple tech rider is included inside the manuscript, and specify the full names/affiliations of the performers/interpreters of the piece.</li>
+  <li><strong>Submitting Large Files</strong> - If the audio/video files are larger than 25MB, please use a file-sharing service of your preference, where a <strong>permanent link</strong> can be provided (in the manuscript) and/or access enabled to a member of the organization team. If the folder is password protected, it is possible to coordinate with the organization so that this password is shared safely.
```

Comment:
```suggestion
  <li><strong>Submitting Large Files</strong> - If the audio/video files are larger than 25MB, please use a file-sharing service of your preference (e.g. Dropbox, Google Drive, etc.), for which a <strong>permanent link</strong> can be provided (in the manuscript), and don't forget to include any passwords that might be needed to access the files.
```

maybe simplify? also, add mention to example hosting services xD

Reply:
Initially I tried to avoid advertising one or another company, but this looks good :)